src/workloads/docs/CrudActorEncrypted.yml

```diff
@@ -65,25 +65,28 @@ Description: |
 GlobalDefaults:
   EncryptedDB: &encrypted_db genny_fle
 
-EncryptedCollections:
-- Database: *encrypted_db
-  Collection: "accounts"
-  EncryptionType: "fle"
-  FLEEncryptedFields:
-    name: { type: "string", algorithm: "random" }
-    credit: { type: "long", algorithm: "random" }
-    pii.ssn: { type: "string", algorithm: "deterministic" }
-    pii.ppn: {type: "string", algorithm: "random"}
-    bio.dob: { type: "date", algorithm: "random" }
-    bio.birthplace.street: { type: "string", algorithm: "random" }
-    bio.birthplace.country: { type: "string", algorithm: "deterministic" }
-- Database: *encrypted_db
-  Collection: "medical"
-  EncryptionType: "fle"
-  FLEEncryptedFields:
-    patient_id: { type: "string", algorithm: "deterministic" }
-    insurance: { type: "string", algorithm: "random" }
-    conditions: { type: "array", algorithm: "random" }
+Encryption:
+  UseCryptSharedLib: true
+  CryptSharedLibPath: /data/workdir/mongocrypt/lib/mongo_crypt_v1.so
```

Comment:
Worth noting that hardcoding this in workload yamls seems a bit risky for breaking things for local users, unless they contort their local filesystem to match.

I wonder if it makes sense to encourage using [overrides](https://github.com/mongodb/genny/blob/master/docs/using.md#org22b7a0f) for the path, so only the caller has to know where the file is. Might be a DSI change worth doing either soon or in the future

Reply:
I commented these out to reduce the risk of breakage. I will explore how these can be overridden in sys-perf via the feature you mentioned.